autospec/download.py

```diff
@@ -44,10 +44,10 @@ def do_curl(url, dest=None, post=None, is_fatal=False):
         c.setopt(c.POSTFIELDS, post)
     c.setopt(c.FOLLOWLOCATION, True)
     c.setopt(c.FAILONERROR, True)
-    c.setopt(c.CONNECTTIMEOUT, 10)
-    c.setopt(c.TIMEOUT, 10)
+    c.setopt(c.CONNECTTIMEOUT, 30)
+    c.setopt(c.TIMEOUT, 30)
     c.setopt(c.LOW_SPEED_LIMIT, 1)
-    c.setopt(c.LOW_SPEED_TIME, 10)
+    c.setopt(c.LOW_SPEED_TIME, 30)
```

Comment:
Similarly, 30s of being 1 byte per second or lower seems a bit much to me too. Given that you moved the absolute timeout to 30s, I don't think this would have an impact either way though.

Reply:
@bryteise I completely agree with your reasoning here.